addon/components/mock-login.hbs

```diff
@@ -0,0 +1,8 @@
+<div ...attributes>
```

Comment:
I think this can be removed in the future. Kept it for now since the Ember component had a tagName.